src/test/java/com/junwoo/ott/domain/video/common/VideoTestCommons.java

```diff
@@ -0,0 +1,18 @@
+package com.junwoo.ott.domain.video.common;
+
+import com.junwoo.ott.domain.video.dto.request.VideoCreateRequestDto;
+import com.junwoo.ott.global.customenum.RatingType;
+
+public interface VideoTestCommons {
```

Comment:
이름을 Values로 통일하는 것은 어떨까요?
```suggestion
public interface VideoTestValues {
```

Reply:
넵!